snapcraft/storeapi/errors.py

```diff
@@ -177,7 +177,9 @@ class StoreRegistrationError(StoreError):
     __FMT_RESERVED = (
         'The name {snap_name!r} is reserved.\n\n'
         'If you are the publisher most users expect for '
-        '{snap_name!r} then please claim the name at {register_name_url!r}')
+        '{snap_name!r} then please claim the name at {register_name_url!r}\n\n'
+        'Otherwise, please register a name that people will associate with '
```

Comment:
To clear out all ambigous language let's just remove the bits that cause contention.

So changing 
```python
'Otherwise, please register a name that people will associate with '
'you. We can rename your snap later if needed.')
```
To just
```python
'Otherwise, please register another name.')
```

should do the trick.